### PR TITLE
Update README.md (contributor count)

### PR DESCRIPTION
**Pull request by siam1113**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 
-[![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors-)
+[![All Contributors](https://img.shields.io/github/contributors/ixahmedxi/noodle)](#contributors-)
```
Comment (xhayper) [inline README.md:9]: Missing `?style=flat-square&color=orange`

**Comment by xhayper** [comment]
From reading this repository's [all-contributor config file](https://github.com/ixahmedxi/noodle/blob/main/.all-contributorsrc) i think it's intended for the follower count to be only 6, since the project owner want to manually add contributor to the list

**Review by brightlooksa — COMMENTED** [review]
hi

**Comment by siam1113** [comment]
> From reading this [repository's all-contributor config file](https://github.com/ixahmedxi/noodle/blob/0e80e7dbc589d054fe50a51a3b47557341ee0866/.all-contributorsrc) i think it's intended for the follower count to be only 6, since the project owner want to manually add contributor to the list

Maybe you right actually

**Comment by LBF38** [comment]
Here is the correct badge to use for all-contributors in GitHub : https://shields.io/badges/git-hub-contributors-via-allcontributors-org

**Comment by LBF38** [comment]
@siam1113 The correct syntax would be the following for the badge link : 
```
https://img.shields.io/github/all-contributors/ixahmedxi/noodle/master?style=flat-square&color=orange
```

**Comment by ixahmedxi** [comment]
@siam1113 we use all-contributors bot, it's my bad for not updating it with every contributor but i will do so to reflect the number of contributors

**Pull request closed without merging**